docs/README.md

```diff
@@ -0,0 +1,14 @@
+# Scan-files Infrastructure
+
+![AWS infrastructure diagram.  Full text description follows.](architecture.png)
+
+1. Virtual Private Cloud (VPC) with three public and three private subnets.
+1. API Gateway (AGW) to allow inbound/outbound communication, protected by Web Application Firewall (WAF), as main user request entry point and has an Lambda API as its target.
+1. NAT Gateways (3) to allow private subnets to communicate with AGW.
```

Comment:
Related to the above question, I didn't realize anything in the private subnets was talking back to the AGW.

Reply:
reworded as well